app/components/views/Home/WalletAction/index.tsx

```diff
@@ -17,13 +23,17 @@ const WalletAction: FC<WalletProps> = ({
       CONNECT WALLET
     </button>
   ) : (
-    <button
-      type="button"
-      className={styles.disconnectWalletButton}
-      onClick={disconnect}
-    >
-      DISCONNECT WALLET
-    </button>
+    <TextInfoTooltip content="DISCONNECT WALLET" singleton={singleton}>
```

Comment:
ah no need for the singleton, thats only when you want to animate/sync up multiple tooltips

Reply:
@Atmosfearful removed it 👍🏻